### PR TITLE
Migrate to WXT

### PR DESCRIPTION
**Pull request by arnab710**
# Tasks

- [x]  Convert from `JS` to `TS`.
- [x]  Migrate to `WXT` from `webpack`.
- [x]  Remove `ExtPay` related code. (@Anshtripathi079 )

### CODE REVIEW

**Comment by arnab710** [comment]
@Anshtripathi079 @GorvGoyl This will be the new file structure.

@Anshtripathi079 I have already removed some code related to ExtPay. Could you please remove all `extPay` related codes in this PR? So, We don't have to deal with 2 different PRs.

**Review by GorvGoyl — CHANGES_REQUESTED** [review]
let's prevent opening a new browser when running `dev` command

**Review by GorvGoyl — COMMENTED** [review]
when running `pnpm run package`, output should be 3 zip files with following names (no need to add version info):
1. notion_boost_chrome.zip
2. notion_boost_firefox.zip
3. notion_boost_sourcecode.zip

**Comment by GorvGoyl** [comment]
Also, the prettier extension is not working properly 
screenshot 2024-08-28 at 13 45 51@2x

File: components/features/openFullPage.ts [hunk 1]
```diff
@@ -87,8 +87,8 @@ function docEditListener() {
       const previewPageID = (fullPageLink.href
         .slice(1)
         .split("&")
-        .map((opt) => opt.split("="))
-        .find((opt) => opt[0] === "p") || [
+        .map((opt:any) => opt.split("="))
+        .find((opt:any) => opt[0] === "p") || [
         "",
         ...fullPageLink.pathname.split(/(-|\/)/g).reverse(),
```
Comment (GorvGoyl) [inline components/features/openFullPage.ts:93]: ts errors in this file: Property 'pathname' does not exist on type 'Element'.

**Pull request merged**